llvm/docs/SPIRVUsage.rst

```diff
@@ -14,10 +14,13 @@ Introduction
 The SPIR-V target provides code generation for the SPIR-V binary format described
```

Comment:
May be a high-level command that will invoke SPIR-V backend and a list of relevant arguments (with one-line descriptions) will be helpful here?

Thanks